packages/node_modules/@ciscospark/react-container-message-composer/src/container.js

```diff
@@ -25,6 +26,7 @@ export class MessageComposer extends Component {
     this.handleTextAreaFocus = this.handleTextAreaFocus.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.setUserTyping = debounce(this.setUserTyping.bind(this), 500, {leading: true, trailing: false});
```

Comment:
How does debounce handle when we pass it different props? Would a `false` immediately send if there was a `true` waiting?

Reply:
Props don't matter, it only checks the function that is called. I think this is fine given that there's only a 500ms debounce.

Reply:
Ok, cool. Didn't want a true to be sent after a false if false was the last one to come in.